api/config/routes.rb

```diff
@@ -11,6 +11,7 @@
       resources :makers
       resources :resources
       resources :texts
+      resources :statistics, only: [:index]
```

Comment:
Seems like this should be a singular resource, not plural, and that your index code should really be in a #show method.